SpatialGDK/Source/SpatialGDK/Private/Interop/SpatialClassInfoManager.cpp

```diff
@@ -215,6 +217,20 @@ void USpatialClassInfoManager::CreateClassInfoForClass(UClass* Class)
 
 		Info->SubobjectInfo.Add(Offset, ActorSubobjectInfo);
 	}
+
+	if (Class->IsChildOf<AActor>())
+	{
+		Info->ActorGroup = ActorGroupManager->GetActorGroupForClass(Class);
```

Comment:
Is this intended just to validate that there are no errors? I don't see any usages of `WorkerType` or `ActorGroup` for now apart from https://github.com/spatialos/UnrealGDK/pull/1059/files#diff-186eee97b9a3627fd25604fa58d7c9d4R132 and we're not sure about that one because it could use `ActorGroupManager` instead. I suggest removing the duplication unless you think it's more efficient to cache this data in the `Info`. If so, could you make the `ActorGroupManager` members only accessible to this class so that others don't get confused about which manager to use?